tests/components/zwave_js/test_update.py

```diff
@@ -311,3 +313,80 @@ async def test_update_entity_failure(
         args["nodeId"]
         == climate_radio_thermostat_ct100_plus_different_endpoints.node_id
     )
+
+
+async def test_update_entity_progress(
+    hass,
+    client,
+    climate_radio_thermostat_ct100_plus_different_endpoints,
+    integration,
+):
+    """Test update entity progress."""
+    node = climate_radio_thermostat_ct100_plus_different_endpoints
+    client.async_send_command.return_value = FIRMWARE_UPDATES
+
+    async_fire_time_changed(hass, dt_util.utcnow() + timedelta(days=1))
+    await hass.async_block_till_done()
+
+    state = hass.states.get(UPDATE_ENTITY)
+    assert state
+    assert state.state == STATE_ON
+    attrs = state.attributes
+    assert attrs[ATTR_INSTALLED_VERSION] == "10.7"
+    assert attrs[ATTR_LATEST_VERSION] == "11.2.4"
+
+    asyncio_event = asyncio.Event()
+
+    async def mock_install_func(node, file):
+        """Mock install function."""
+        await asyncio_event.wait()
+
+    with patch(
+        "homeassistant.components.zwave_js.update.Controller.async_begin_ota_firmware_update",
```

Comment:
Probably set the side effect on `client.async_send_command` instead.

Reply:
no longer relevant with the change to manage the Event in the platform